spec/subscribe_integration_spec.rb

```diff
@@ -9,6 +9,8 @@
       @product = Z::Product.where(:name => "Awesome Product").first ||
         Z::Product.create!(
           :name => "Awesome Product",
+          :features__c => "Now with even more awesome features!",
```

Comment:
I like the Idea of testing custom fields but I don't think we should do this. It makes it harder for others to run the tests against their accounts.